app/src/app/components/view-profile/view-profile.component.html

```diff
@@ -0,0 +1,186 @@
+<div class="profile_container">
```

Comment:
incorrect naming convention for the class names, should be kebab style and start with the `aca-` or `app-` prefix, please correct those

Reply:
Done.